main.py

```diff
@@ -422,9 +439,7 @@ def __init__(
         self.rescale = rescale
         self.batch_freq = batch_frequency
         self.max_images = max_images
-        self.logger_log_images = {
-            pl.loggers.TestTubeLogger: self._testtube,
-        }
+        self.logger_log_images = { pl.loggers.TestTubeLogger: self._testtube, } if torch.cuda.is_available() else { }
```

Comment:
TestTubeLogger was removed in newer py lighting, so this is not a cuda specific thing

Reply:
Yes, but we are using `pytorch-lighting==1.7.5` to train on the GPU, while `environment.yaml` has `pytorch-lightning==1.4.2`. So I'd assume TestTubeLogger works for non-MPS devices (?).

Reply:
Okay if they're using a different environment and version then this makes sense :+1:

Reply:
@lstein You can change this for cuda